softwareComponents/configuration/include/configuration/rofibot.hpp

```diff
@@ -550,20 +550,24 @@ class Rofibot {
     }
 
     /**
-     * \brief Decided whether the configuration is valid
+     * @brief Return true if the configuration is prepared
+     * 
+     * Configuration can be prepared with `prepare`.
+     */
+    bool isPrepared() const {
+        return _prepared;
+    }
+
+    /**
+     * \brief Decide whether the configuration is valid
      *
      * \return A pair - first item indicates the validity, the second one gives
      * textual description of the reason for invalidity
      */
     template < typename Collision >
```

Comment:
Consider adding a default collision type (e.g. SimpleCollision).

Reply:
This is already part of the `configuration` branch (namely here 482e022538100b4944555d8daa41752150c12d13), so it will propagate to master eventually.